app/src/organisms/ProtocolSetupLabware/index.tsx

```diff
@@ -354,15 +354,12 @@ function LabwareLatch({
       >
         {statusText != null && icon != null ? (
           <>
-            <StyledText
-              fontSize="1.375rem"
-              fontWeight={TYPOGRAPHY.fontWeightRegular}
-            >
+            <StyledText fontWeight={TYPOGRAPHY.fontWeightRegular}>
               {t(statusText)}
             </StyledText>
             <Icon
               name={icon}
-              size="2.5rem"
+              size={SPACING.spacingXXL}
```

Comment:
let's avoid using spacing for size/width/height for now.
Mel will be in the Dumbo office, so we might add constants for them.

```suggestion
              size="2.5rem"
```